src/themes/partials/variables/_dark-variables.scss

```diff
@@ -0,0 +1,7 @@
+$dark: #222222;
```

Comment:
Do a check of the variables. The dark theme looks like a light theme with a mix of dark backgrounds. Please verify this.